### ISSUE TITLE
[User Feedback] Add Option to Hide Feedback Messages on Issue Details & Persist That As a User Setting

### ISSUE THREAD

**Issue by jas-kas** [getsentry/sentry#74209]
## Problem

TL/DR: Issue Details page can show “User Feedback” section when the feedback comes from a crash report. 

Sometimes the feedback is vulgar and Sentry users don’t want to see that.

## Solution

Task: Make the feedback collapsable, and remember server-side that feedback is collapsed/hidden. The setting is applied to any feedback rendered directly on the Issue Details page, so if people hide the feedback once there, it’s hidden all the time for all feedbacks on Issue Details. Note: feedbacks on the the /feedback page and the Issue Details > User Feedback tab will still be visible as they are dedicated views for solely feedback messages.

┆Issue is synchronized with this [Jira Improvement](https://getsentry.atlassian.net/browse/FEEDBACK-2222) by [Unito](https://www.unito.io)

**Comment by jas-kas** [comment]
cc/ @ryan953 @aliu39 @michellewzhang

**Comment by michellewzhang** [comment]
similar to what we did for the issue details onboarding banner:

- [ ] BE: add a new prompt config to the backend: ex. https://github.com/getsentry/sentry/pull/69513/files
- [ ] FE: add a chevron to the user feedback component in `userFeedback.tsx` that will allow the user to collapse/expand the feedback (maybe add a message like "feedback is hidden" when it's hidden)
- [ ] FE: remember the hidden state using the `usePrompt` hook on a **PROJECT** level: ex. https://github.com/getsentry/sentry/pull/69525/files
